### ISSUE TITLE
lit-element doesn t allow return of only a slot element inside the render()

### ISSUE THREAD

**Issue by andreabosman16** [Polymer/lit-element#189]
When only returning html` <slot></slot>` element in the render() function
the output in html gives only a empty shadow dom and all the elements are placed inside the light dom.

When adding a div element and a property to the returning html the shadow dom is filled as expected.
is this a bug or expected behaviour ?

**Comment by aadamsx** [comment]
Will you post this on CodePen or jsbin?

**Comment by andreabosman16** [comment]
how can i use lit-element inside codepen i never used it before so if you could assist me that would be great

**Comment by abraham** [comment]
There are several hosted code sandboxes listed in the readme that you can use as a starting spot.

https://github.com/Polymer/lit-element#getting-started

**Comment by jsilvermist** [comment]
Seems to work fine to me.

https://glitch.com/edit/#!/lit-element-base-slot?path=index.html

**Comment by andreabosman16** [comment]
The  example provided by @jsilvermis is working fine. I don t get it I am doing the same thing but with an external file, could it be web pack and typescript? Another thing I would like to discuss your documentation is up to date but the application on npm isn't the version on npm needs _ before the render function, and the GitHub version without the _. this is the same inside the jsbin and codepen examples.

**Comment by jsilvermist** [comment]
For now, you can install the latest version using:
```bash
npm install @polymer/lit-element@dev
```
which currently resolves to @polymer/lit-element@^0.6.0-dev.6.

**Comment by jsilvermist** [comment]
Not exactly sure what you mean there, but if you mean toggling a boolean attribute, you can use the following syntax:
```html
<some-el ?attr=${someBooleanOrExpression}></some-el>
```
or in the case of something like `checked` on a checkbox, you should set the boolean as a property to ensure proper updates, as in:
```html
<input type="checkbox" .checked=${someBooleanOrExpression}>
```
If these aren't correct, please add an example of what you're trying to achieve.

**Comment by jsilvermist** [comment]
```js
<input ?readonly=${read_only}></input>
```

**Comment by andreabosman16** [comment]
`<input ?readonly=${read_only}></input>`

this doesn't t work because of it not being a boolean. When the readonly property is set inside the input element it will always be readonly.
So if the boolean changes it will not change between readonly or writeable

I found the error with the question I asked 

this was my code:

  `connectedCallback(){
         console.log(${this.id})
    };`

` render(){
        console.log("render called on Hbox");
        return html<slot></slot>;     
    }`

when I used the `connectedCallback` the render wasn t working properly. When I removed the `connectedCallback` everything worked fine. Is this expected behavior or not?
